schema/rpc.fbs

```diff
@@ -541,4 +539,16 @@ table SaveFileNotification {
     expected_dir: ComputerDirectory = null;
     /// Recommended filename
     expected_filename: string;
+    /// ID of the SaveFile, needs to be returned by the SaveFileRequest
+    id: uint32;
+}
+
+/// Response of the SaveFileNotification after the user interacts with the file save request
+table SaveFileRequest {
```

Comment:
If this is a response, why is it named request?

Reply:
Dk if to call Response calls that come from only the server or if they can also come from the client

Reply:
I thought the naming we were using was this:

Request: Something that has 0-1 responses
Response: Always corresponds to a particular request
Notification: This does not correspond to any particular response.